WordPress/src/main/res/values/strings.xml

```diff
@@ -1049,6 +1049,10 @@
 
     <!-- backup -->
     <string name="backup">Backup</string>
+    <string name="backup_empty_title">Your first backup will be ready soon</string>
+    <string name="backup_empty_subtitle">Your first backup will appear here within 24 hours and you will receive a notification once the backup has been completed</string>
```

Comment:
> Your first backup will appear here within 24 hours 

❓ I believe that's the case for "BackupDaily", are we sure it's the case even for other products/plans?

Reply:
Hmmm... I think you are right! 🌟

Adding @osullivanchris and @leandroalonso to this thread to increase transparency and maybe reach to a conclusion faster.

Your thoughts folks?

PS: In the meanwhile, I'll try and find what Calypso is displaying in such cases.

Reply:
Is the question whether that 'first backup will appear here in 24 hours' applies in other cases? If so, to be honest, I don't really know the differences in enough detail to know the answer here! For real-time backups, it could appear even sooner, but within 24 hours is still true. Are there some customers who will never get backups, or get them later than 24 hours?

Reply:
> Are there some customers who will never get backups, or get them later than 24 hours?

If we're showing this screen, the blog has the backups feature so the user will either have a backup in 24h or soon.

We can check the capabilities and change this message based on it (that way you can tell if it's `realtime` or `daily`).

I just copied the logic/message from the web, I'm really not sure if it worths it to add that logic just to change a message. If the user just bought the real-time backup it will probably start working right away, so the results will be there.

Reply:
If it can't happen at a later time than in 24hours, I incline towards not changing it ;)! I thought we support even one-time backups (BACKUP capability), but perhaps it means something else.

Reply:
> For real-time backups, it could appear even sooner, but within 24 hours is still true.

> If the user just bought the real-time backup it will probably start working right away, so the results will be there.

This actually makes sense, considering the above I would let is be as it is now.

@malinajirka your thoughts?

PS: Thanks for jumping in @osullivanchris @leandroalonso ! ❤️